src/car.cpp

```diff
@@ -66,27 +63,19 @@ bool Salmon::init()
 		return false;
 
 	// Loading shaders
-	if (!effect.load_from_file(shader_path("colored.vs.glsl"), shader_path("colored.fs.glsl")))
+	if (!effect.load_from_file(shader_path("colored.vs.glsl"), shader_path("colored.fs.glsl"))) // TODO: Change shaders?

```

Comment:
If we're just using a flat texture for the car and stuff, then I don't think we need to change the shaders. We'll probably need to make changes if we start doing more advanced graphical stuff (lighting/weather effects/etc.)?